torch/_prims_common/__init__.py

```diff
@@ -1494,11 +1494,17 @@ def make_contiguous_strides_for(
     if not shape:
         return ()
 
+    def _ensure_at_least(a, thresh):
+        # sym_max() isn't supported for SingletonSymNode
+        if hasattr(a, "node") and a.node.singleton_int() is not None:
```

Comment:
nit:
```suggestion
        if isinstance(a, torch.SymInt) and a.node.singleton_int() is not None:
```
hmmm interesting why its still going to the other path, you might want to check if threshold is also a singleton int?

Reply:
Discussed offline: we need to check for SymInt wrapping a SingletonSymNode as well (this happens for the fake-ified case)